Assignment_1/Assignment_1/main.swift

```diff
@@ -0,0 +1,145 @@
+    }
+}
+
+if weapon == "axe" {
+    switch fight {
+    case "fight":
+        sleep(1)
+        print("Relying fully on insticts you grab your axe and block their strike. Your opponent is quick but weaker than you. Realizing this you decide to wait until they show an opening in their attacks. As you parry and block the incoming strikes you see an opening to attack...")
+    case "run":
+        sleep(2)
+        print("You attempt to run away but your opponent catches up and strikes you. As you plead for mercy from your enemy, they slowly walk towards your immoblie body and finish you off... coward!")
+    default:
+        print("dead")
+    }
+    
+    print()
+    sleep(5)
+    
+    if randomNumber >= 15 && fight != "run"{
+        sleep(1)
+        print()
+        print("💥 Critical Strike! You rolled a \(randomNumber)! You have defeated your opponenet and have won your first Mak'gora!")
+    } else if randomNumber <= 15 && fight != "run"{
+        sleep(2)
+        print()
+        print("You rolled a \(randomNumber). You have weakened your enemy but have yet to defeat them. Roll again quickly!")
+        randomNumber -= -1
+        sleep(3)
+        print()
+        print("You re-rolled a \(randomNumber)! You have slain the your opponent in 2 strikes! You have won your first Mak'gora!")
+    }
+}
+
+if weapon == "bow and arrow" {
+    switch fight {
+    case "fight":
+        sleep(1)
+        print("You and your hunting party quickly dodge the cyclops and hide behind the trees and rocks in the forest. You weild your bow and arrow, thinking of a way to take your new enemy down. The cyclops is dumb and slow and realize shooting his eye will buy you and your party enough time to escape. You find an opening and shoot!...")
+    case "run":
+        sleep(1)
+        print("Why would you try to run from a giant cyclops?! Those dudes are mad fast and you just died... coward!")
+    default:
+        print("dead")
+    }
+    
+    print()
+    sleep(5)
+    
+    if randomNumber >= 15 && fight != "run"{
+        sleep(1)
+        print("💥 Critical Strike! You rolled a \(randomNumber)! You have defeated the mighty cyclops! You retrieve the game you've hunted and procced to leave the forest and return to your village.")
+    } else if randomNumber <= 15 && fight != "run"{
+        sleep(2)
+        print("You rolled a \(randomNumber). You fire your arrow but miss your mark! Roll again quickly!")
+        randomNumber -= -1
+        print()
+        sleep(3)
+        print("You re-rolled a \(randomNumber)! You have finally slain the the mighty cyclops! You retrieve the game you've hunted and procceed to leave the forest and return to your village.")
+    }
+}
+
+print()
+print()
+sleep(1)
+print("Would you like to play again?")
+
+let decision = readLine()?.lowercased()
+switch decision {
+case "yes":
+    print("Then hit ⌘ R!")
```

Comment:
lol that's a great use of having the player playing again. I never have seen that before that's really smart. You can have the player play again using a while loop so if they say "yes" it would loop to the very beginning of your code